tests/TestCase/Database/Schema/MysqlSchemaTest.php

```diff
@@ -410,6 +410,9 @@ public function testDescribeTable()
         if (ConnectionManager::get('test')->getDriver()->isMariadb()) {
             $expected['created_with_precision']['default'] = 'current_timestamp(3)';
             $expected['created_with_precision']['comment'] = '';
+            if (!empty($expected['collate'])) {
+                $expected['collate'] = 'utf8mb3_general_ci';
+            }
```

Comment:
```suggestion
            $expected['title']['collate'] = 'utf8mb3_general_ci';
            $expected['body']['collate'] = 'utf8mb3_general_ci';
```

🤔

Reply:
Whoops, my change wouldn't have worked at all. Thank you :bow: